cartographer_rviz/cartographer_rviz/drawable_submap.cc

```diff
@@ -159,13 +168,16 @@ void DrawableSubmap::SetAlpha(const double current_tracking_z) {
 
 void DrawableSubmap::UpdateSceneNode() {
   ::cartographer::common::MutexLocker locker(&mutex_);
+  render_in_progress_ = false;
+  if (quitting_) {
```

Comment:
Yes, sounds good. I was only commenting on the second commit, the first one can be merged as is.